session/session.go

```diff
@@ -1212,6 +1211,7 @@ func (s *session) execute(ctx context.Context, sql string) (recordSets []sqlexec
 	multiQuery := len(stmtNodes) > 1
```

Comment:
This function is not used, or won't be used  in the future.
`Execute()` returning multiple record sets are deprecated.

- The normal query runs through ExecuteStmt
- Prepared query runs through ExecutePreparedStmt
- Only internal SQLs are still running to here, and I'm cleaning up the code.